AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPy/QtPyAssetEditor.py

```diff
@@ -78,6 +141,23 @@ def update_new_method_name(self, old_method_name: str, updated_method_name: str)
         assert old_method_exists, f"Failed to find method {old_method_name} in Asset Editor."
         assert method_name_field == updated_method_name, "Failed to set method name in Asset Editor."
 
+    def delete_parameter_from_method(self, parameter_index: int) -> None:
+        """
+        function for deleting a parameter from a script event. This will delete the parameter at index 0.
+        This function will need to be updated to support
```

Comment:
Incomplete thought in the docstring here. If there's upcoming work planned for these, maybe include a link to the ticket for the task for tracking.

Reply:
Arg. This was actually updated to support multiple params. I jsut forgot to delete the docstring.